WooCommerce/WooCommerceTests/POS/Presentation/StockFormatterTests.swift

```diff
@@ -0,0 +1,114 @@
+import Foundation
+import Testing
+import struct Yosemite.POSSimpleProduct
+@testable import WooCommerce
+
+struct StockFormatterTests {
+    @Test func test_managestock_when_disabled_and_stockStatusKey_not_set_then_returns_empty_stockLabel() async throws {
```

Comment:
nit: it might sound like it's testing "managestock" (same for other test cases)
```suggestion
    @Test func test_when_manageStock_disabled_and_stockStatusKey_not_set_then_returns_empty_stockLabel() async throws {
```